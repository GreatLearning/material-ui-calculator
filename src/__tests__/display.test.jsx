import renderer from 'react-test-renderer';
import React from 'react';
import Display from '../display';


it('renders correctly', () => {
  const tree = renderer
    .create(<Display showAngleMode={true} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
