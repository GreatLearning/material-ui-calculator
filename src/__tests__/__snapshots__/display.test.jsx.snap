// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="Display-display-2"
>
  <div
    className="Display-angleMode-1"
  >
    <div
      className="Component-mode-6"
      onClick={[Function]}
    >
      deg
    </div>
    <div
      className="Component-mode-6"
      onClick={[Function]}
    >
      rad
    </div>
  </div>
  <div
    className="Display-expr-5"
  >
     
  </div>
</div>
`;
